Sources/Layout/MessageSizeCalculator.swift

```diff
@@ -44,11 +44,21 @@ open class MessageSizeCalculator: CellSizeCalculator {
     public var incomingCellTopLabelAlignment = LabelAlignment(textAlignment: .center, textInsets: .zero)
     public var outgoingCellTopLabelAlignment = LabelAlignment(textAlignment: .center, textInsets: .zero)
 
-    public var incomingMessageTopLabelAlignment = LabelAlignment(textAlignment: .left, textInsets: UIEdgeInsets(left: 42))
-    public var outgoingMessageTopLabelAlignment = LabelAlignment(textAlignment: .right, textInsets: UIEdgeInsets(right: 42))
+    public var incomingMessageTopLabelAlignment: LabelAlignment {
```

Comment:
The problem with these now being stored properties is that they are no longer configurable

Reply:
Yep you're right. I was thinking about subclassing (MessageSizeCalculator or another subclass of it) to override this values, but I forget to modify `public` to `open`.